code/addons/storysource/src/StoryPanel.tsx

```diff
@@ -112,9 +112,11 @@ export const StoryPanel: React.FC<StoryPanelProps> = ({ api }) => {
   };
 
   const createParts = ({ rows, stylesheet, useInlineStyles }: SyntaxHighlighterRendererProps) => {
-    const parts = [];
+    const parts: any[][] = [];
```

Comment:
Why `any`? The return value is being used as a `children` prop, so why not `React.ReactNode` or `React.ReactElement`?

Reply:
@ryami333 Yeah, my bad. I just wasn't sure what the type of `parts` really was since I couldn't figure out what `createElement` returns. I'll address this asap.